src/components/calcite-value-list/calcite-value-list.tsx

```diff
@@ -265,21 +276,29 @@ export class CalciteValueList {
     return type;
   }
 
+  renderScrim(): VNode {
+    return this.loading || this.disabled ? (
+      <CalciteScrim loading={this.loading}></CalciteScrim>
+    ) : null;
+  }
+
   render() {
+    const { dataForFilter, handleFilter, disabled, filterEnabled, loading } = this;
     return (
-      <Host>
+      <Host aria-busy={loading}>
```

Comment:
I want to move this attr out of Host.
Would a `div` be an appropriate wrapper inside Host?
@jcfranco @pr3tori4n @driskull

Reply:
Yes, that would be sufficient.

Reply:
Thanks!